cristin-project/src/test/java/no/unit/nva/cristin/projects/ContributorRoleMappingTest.java

```diff
@@ -3,27 +3,29 @@
 import no.unit.nva.utils.ContributorRoleMapping;
 import org.junit.jupiter.api.Test;
 
+import java.util.NoSuchElementException;
+
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class ContributorRoleMappingTest {
 
     @Test
     void shouldReturnManagerMapping() {
-        assertTrue(ContributorRoleMapping.getNvaRole("PRO_MANAGER").equals("ProjectManager"));
-        assertTrue(ContributorRoleMapping.getCristinRole("ProjectManager").equals("PRO_MANAGER"));
+        assertTrue(ContributorRoleMapping.getNvaRole("PRO_MANAGER").get().equals("ProjectManager"));
```

Comment:
These tests are not needed as the code should be touched directly from the handler